src/main/kotlin/no/nav/familie/ef/sak/økonomi/domain/AndelTilkjentYtelse.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.familie.ef.sak.økonomi.domain
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Column
+import java.time.LocalDate
+
+data class AndelTilkjentYtelse(
```

Comment:
Fungerer dette uten @Table(andel_tilkjent_ytelse)

Reply:
Ja. Se tilsvarende for TilkjentYtelse